### DISCUSSION TITLE
Gestion de l'inventaire

### DISCUSSION THREAD

**Discussion by Sousoulman** [Crownicles/Crownicles#2355] [Nouvelles idées]
Pour moi, il devrait être possible d'intervertir un objet depuis l'inventaire jusqu'à la réserve même si la réserve est vide car on ne veut pas forcément utiliser une potion durant un combat on ne peut pas l'échanger et si on veut vendre un item mais que la réserve et vide et bien il fait attendre que l'on reçoivent un item pour l'échanger pour pouvoir vendre l'item
(Prévenez-moi si il y a déjà un moyen et que je ne le connais pas)

**Comment by Voisin1234** [comment]
D’accord

**Comment by Voisin1234** [comment]
Genre on peux même pas rager et tout vendre sur un coup de tête ! 🤭
